apps/omg_performance/test/omg_performance/byzantine_event_test.exs

```diff
@@ -47,6 +47,17 @@ defmodule OMG.Performance.ByzantineEventsTest do
     |> ByzantineEvents.get_many_standard_exits()
   end
 
+  @tag fixtures: [:perf_test, :mix_based_child_chain, :mix_based_watcher]
+  test "can provide timing of response when asking for IFE data", %{perf_test: {:ok, %{destdir: destdir}}} do
```

Comment:
what does the test trying to prove? 
`can provide timing of response when asking for IFE data`

Reply:
so the tooling under test is aimed to (among others) provide timing of stuffs (you execute a command and measure time it takes, somehow). So if code in this test works, you're able to do the "provide timing" part, that's where the title is coming from. Considering this, if this wording is confusing to you, would something simpler like `can ask for many IFE data` work better? This alternative is more in line with what the test actually runs, less in line with how the tested tool is intended to be used.